doc/source/dev/releasing.rst

```diff
@@ -142,7 +146,10 @@ For a release there are currently five places on the web to upload things to:
 
 **PyPI:**
 
-``twine upload -s <tarballs or wheels to upload>``
+Upload first the wheels and then the sdist::
+
+  twine upload -s PATH_TO_STORE_WHEELS/*.whl
```

Comment:
Should make sure the downloaded wheels for other versions are gone.

Reply:
Good point. Changed to ``REPO_ROOT/release/installers``, old wheels cannot be present because a ``git clean -xdf`` is performed before generating sdist.